aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -261,7 +262,7 @@ void apply_linalg_eig(Tensor& values, Tensor& vectors, Tensor& input, Tensor& in
     scalar_t* rvectors_working_ptr = compute_eigenvectors ? &rvectors_data[i * input_matrix_stride] : nullptr;
     int* info_working_ptr = &infos_data[i];
     lapackEig<scalar_t, value_t>(jobvl, jobvr, n, input_working_ptr, lda, values_working_ptr,
-      lvectors_data, ldvl, rvectors_working_ptr, ldvr, work_data, lwork, rwork_data, info_working_ptr);
+                                 lvectors_data, ldvl, rvectors_working_ptr, ldvr, work_data, lwork, rwork_data, info_working_ptr);
```

Comment:
Changes in this file are not related, they can be reverted with `git checkout master -- aten/src/ATen/native/BatchLinearAlgebraKernel.cpp`